CHANGELOG.rst

```diff
@@ -80,6 +80,11 @@ TBR
 * Moved some of the utility functions from the test module into
   ``scrapy_poet.utils.testing``.
 
+* Now requires Twisted >= 21.7.0 instead of >= 18.9.0. This prevents a ``TypeError``
+  from happening since scrapy-poet uses the ``twisted.internet.defer.Deferred[object]`` 
```

Comment:
A more conservative fix would be to not use this type annotation. Is it essential?

Reply:
Yes, I think that's a better solution.

I'll make this change instead in #129 since it adds another type annotation using `Deferred`. Commit: efb6476